frontend/src/AlertsManagement/AlertsContent.js

```diff
@@ -45,30 +45,36 @@ TabPanel.propTypes = {
   value: PropTypes.any.isRequired,
 };
 
-// TODO: fetch alerts from backend and save in arrays.
-// Intended plan for reference: 
-// const allAlerts = [id1, id2, id3] where id'x' is the id of AlertX
-// const alertsMap = {id1: <Alert1>, 2: <Alert2>, 3: <Alert3>, ...};
-// const unresolvedAlerts = [0, 1] which stores the indices of the alerts in allAlerts
-// const resolvedAlerts = [2]
-const allAlerts = [0, 1, 2, 3, 4, 5, 6];
-const unresolvedAlerts = [0, 1, 2, 3];
-const resolvedAlerts = [4, 5, 6]
-
 // TODO: create constants file. ref: https://stackoverflow.com/questions/39036457/react-create-constants-file.
 const tabLabels = {
   UNRESOLVED: 0,
   RESOLVED: 1,
   ALL: 2,
 };
 
+/*
+ * Helper function to create Date from Timestamp model.
+ */
+const createDate = (timestampDate) => {
```

Comment:
could we also have a time utility file for time/date-related utility functions since i assume there will be a need for this to be in sync across various components and make it clear on the logic vs ui-related code.

other things that might be included in this time/date-related utility file might come in later as you add new features, but good to have things clean in the beginning to reduce refactoring later on.

Some example of a date/time utility file in js: https://docs.google.com/document/d/1XcSTkWcDy4OAIwI16BhkOq2XcVK0Up28EL2Qls9Fqso/edit?hl=en. Feel free to comment directly on the doc/ask if you have any q's.